### DISCUSSION TITLE
Incredibly slow OpenCL build process

### DISCUSSION THREAD

**Discussion by SleepiDreamer** [MathIsFun0/Immolate#38] [Q&A]
with a "simple" 100-line OpenCL kernel I get build times of up to 5 minutes on a Ryzen 5 7600X. Is this expected? This makes quick iteration impossible, so I was wondering if there's anything I can do about this...

**Comment by OptimusPi** [comment]
Did you ever find anything out?

So far I just have the placebo of Ctrl+Shift+WindowsKEy+B to reboot the OpenCL driver lol

**Comment by OptimusPi** [comment]
Check out this branch of my fork: https://github.com/OptimusPi/Immolate/tree/ouiji

**Reply by OptimusPi** [reply]
supports binary caching so it's not slow to recompile the opencl program, if the kernel .bin file is present :D
